CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ project(fpga-interchange)
 
 set(RAPIDWRIGHT_PATH ${PROJECT_SOURCE_DIR}/third_party/RapidWright CACHE PATH "Path to RapidWright")
 set(INVOKE_RAPIDWRIGHT "${RAPIDWRIGHT_PATH}/scripts/invoke_rapidwright.sh" CACHE PATH "Path to RapidWright invocation script")
+set(JAVA_HEAP_SPACE "-Xmx8g" CACHE STRING "Heap space reserved for Java")
```

Comment:
Not sure about stock GH runners but I'd set it to some larger value (eg. `16g`?)  For `200t` 8GB may be a bit too close to the minimum required.

I've recorded ~9.4GB peak RSS for RapidWright on a 32GB RAM machine but that was without `-Xmx` option provided. Not sure how Java sets the max heap size in that case.

Reply:
Right, I'll increase this

Reply:
Increased to 32G